### PR TITLE
Update Editor Config

### PR DESCRIPTION
**Pull request by solonovamax**
# Pull Request

## Brief description.

Update the editor config to better reflect current java standards

## Licensing

- [x] I am the original author of this code, and I am willing to release it under [GPLv3](https://www.gnu.org/licenses/gpl-3.0.en.html).
- [ ] I am not the original author of this code, but it is in public domain or released
  under [GPLv3](https://www.gnu.org/licenses/gpl-3.0.en.html) or a compatible license.
    

## Goal of the PR

- [x] Make code style better reflect modern java standards

#### Types of changes

- [ ] Bug Fix 
- [ ] Build system 
- [ ] Documentation 
- [ ] New Feature 
- [ ] Performance 
- [ ] Refactoring 
- [ ] Repository 
- [ ] Revert 
- [x] Style 
- [ ] Tests 
- [ ] Translation 

#### Compatiblity

- [ ] Breaking change 
- [x] Non-Breaking change.
    

#### Contribution Guidelines.

- [x] I have read the [`CONTRIBUTING.md`](https://github.com/PolyhedralDev/Terra/blob/master/CONTRIBUTING.md) document in the root of the
  git repository.
- [x] My code follows the code style for this
  project. 

#### Documentation

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

#### Testing

- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: common/src/main/java/com/dfsek/terra/addon/AddonClassLoader.java [hunk 1]
```diff
@@ -13,49 +13,50 @@
...
     @SuppressWarnings("unchecked")
-    public static Set<Class<? extends TerraAddon>> fetchAddonClasses(File file) throws IOException {
-        JarFile jarFile = new JarFile(file);
-        Enumeration<JarEntry> entries = jarFile.entries();
-
-        AddonClassLoader loader = new AddonClassLoader(new URL[] {file.toURI().toURL()}, AddonClassLoader.class.getClassLoader());
-
-        Set<Class<? extends TerraAddon>> set = new HashSet<>();
+    public static Set<Class<? extends TerraAddon>> fetchAddonClasses(final File file) throws IOException {
+        final JarFile jarFile = new JarFile(file);
```
Comment (dfsek) [inline common/src/main/java/com/dfsek/terra/addon/AddonClassLoader.java:32]: Turn off whatever is doing this (the `final` spam).

File: common/src/main/java/com/dfsek/terra/addon/AddonClassLoader.java [hunk 2]
```diff
@@ -13,49 +13,50 @@
...
                 if(addon == null) continue;
-
+                
                 if(!TerraAddon.class.isAssignableFrom(clazz))
                     throw new IllegalArgumentException("Addon class \"" + clazz + "\" must extend TerraAddon.");
-
+                
                 set.add((Class<? extends TerraAddon>) clazz);
-            } catch(ClassNotFoundException e) {
+            } catch(final ClassNotFoundException e) {
```
Comment (dfsek) [inline common/src/main/java/com/dfsek/terra/addon/AddonClassLoader.java:55]: And this (`final` spam on `catch`)

File: common/src/main/java/com/dfsek/terra/addon/AddonPool.java [hunk 3]
```diff
@@ -7,35 +7,36 @@
 import java.util.Map;
 import java.util.Set;
 
+
 public class AddonPool {
     private final Map<String, PreLoadAddon> pool = new HashMap<>();
-
-    public void add(PreLoadAddon addon) throws AddonLoadException {
+    
+    public void add(final PreLoadAddon addon) throws AddonLoadException {
```
Comment (dfsek) [inline common/src/main/java/com/dfsek/terra/addon/AddonPool.java:14]: And this. (`final` spam on method parameters)

**Review by dfsek — CHANGES_REQUESTED** [review]

File: common/src/main/java/com/dfsek/terra/api/event/TerraEventManager.java [hunk 4]
```diff
@@ -20,89 +20,94 @@
...
-                    } catch(Exception e) {
-                        StringWriter writer = new StringWriter();
-                        e.printStackTrace(new PrintWriter(writer));
-                        main.logger().warning("Exception occurred during event handling:");
-                        main.logger().warning(writer.toString());
-                        main.logger().warning("Report this to the maintainers of " + listenerHolder.method.getName());
-                    }
-                }
-        );
+                                                                                      try {
```
Comment (dfsek) [inline common/src/main/java/com/dfsek/terra/api/event/TerraEventManager.java:35]: why on earth is this so far over    
it was fine the way it was

File: common/src/main/java/com/dfsek/terra/api/math/noise/samplers/ImageSampler.java [hunk 5]
```diff
@@ -5,66 +5,73 @@
...
             }
-        }, GRAYSCALE {
+        },
+        GRAYSCALE {
             @Override
             public int getChannel(int mashed) {
                 return (RED.getChannel(mashed) + GREEN.getChannel(mashed) + BLUE.getChannel(mashed)) / 3;
             }
-        }, ALPHA {
+        },
```
Comment (dfsek) [inline common/src/main/java/com/dfsek/terra/api/math/noise/samplers/ImageSampler.java:67]: TBH i much prefer these on the same line

**Comment by solonovamax** [comment]
Going to close this and reopen it with a cleaner history

**Pull request closed without merging**